docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md

```diff
@@ -1,40 +1,40 @@
 ---
-title: Install and use a NuGet package using the dotnet CLI
-description: A walkthrough tutorial on the process of installing and using a NuGet package in a .NET Core project.
+title: Install and use a NuGet package with the dotnet CLI
+description: Get a quick tutorial on how to use the dotnet CLI to install and use a NuGet package in a .NET project.
 author: JonDouglas
 ms.author: jodou
-ms.date: 01/23/2018
+ms.date: 08/29/2022
 ms.topic: quickstart
 ---
 
-# Quickstart: Install and use a package using the dotnet CLI
+# Quickstart: Install and use a package with the dotnet CLI
 
-NuGet packages contain reusable code that other developers make available to you for use in your projects. See [What is NuGet?](../What-is-NuGet.md) for background. Packages are installed into a .NET Core project using the `dotnet add package` command as described in this article for the popular [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) package.
+NuGet packages contain compiled binary code that developers make available for other developers to use in their projects. For more information, see [What is NuGet](../What-is-NuGet.md). This quickstart describes how to install the popular [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json) NuGet package into a .NET project by using the [dotnet add package](/dotnet/core/tools/dotnet-add-package) command.
 
-Once installed, refer to the package in code with `using <namespace>` where \<namespace\> is specific to the package you're using. You can then use the package's API.
+You refer to installed packages in code with a `using <namespace>` directive, where `<namespace>` is the package name. You can then use the package's API in your project.
```

Comment:
```suggestion
You refer to installed packages in code with a `using <namespace>` directive, where `<namespace>` is often the package name. You can then use the package's API in your project.
```

There's nothing preventing a package's namespace to be different from the package id. Maybe I'm just being overly pedantic.